src/collective/cover/static/js/layout_edit.js

```diff
@@ -1,4 +1,112 @@
 (function($) {
+  var CSSClassWidget = (function() {
```

Comment:
how are we going to deal with intermediate caching servers and this change?

Reply:
the simplest solution in this case is to rename the resource name appending the modify date.

Reply:
this needs an upgrade step since this file is into js registry